Makefile

```
@@ -17,3 +17,34 @@ package:
 
 clean:
 	snapcraft clean
+	rm -r .flatpak-builder build/
+
+update-emojis:
+	wget -O static/emojis.csv https://raw.githubusercontent.com/hfg-gmuend/openmoji/master/data/openmoji.csv
+
+flatpak:
+	flatpak-builder --user --install --force-clean build flatpak/com.tomjwatson.Emote.yml
+	flatpak run com.tomjwatson.Emote
+
+flatpak-install:
+	flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
+	flatpak install flathub -y org.flatpak.Builder org.gnome.Platform//43 org.gnome.Sdk//43 org.freedesktop.appstream-glib
+	wget -N https://raw.githubusercontent.com/flatpak/flatpak-builder-tools/master/pip/flatpak-pip-generator
+	chmod +x flatpak-pip-generator
+
+flatpak-requirements:
+	pipenv lock
+	pipenv requirements > requirements.txt
+	pipenv run flatpak-pip-generator --runtime='org.gnome.Sdk//43' --output python3-requirements -r requirements.txt
```

```suggestion
	pipenv run ./flatpak-pip-generator --runtime='org.gnome.Sdk//43' --output python3-requirements -r requirements.txt
```

For me I get this error without this change

```
Error: the command flatpak-pip-generator could not be found within PATH or Pipfile's [scripts].
```